server/app/services/pre_call_sms_worker.py

```
@@ -0,0 +1,536 @@
+                self.key = key
+                self.zone = timezone.utc
+            elif key == "Asia/Kolkata":
+                self.key = key
+                # Indian Standard Time (IST) is UTC+5:30
+                self.zone = timezone(timedelta(hours=5, minutes=30))
+            else:
+                logging.warning(f"ZoneInfo fallback: using UTC for unknown timezone {key}")
+                self.key = key  
+                self.zone = timezone.utc
+        
+        def __call__(self, dt):
```

Ruff (0.14.8)

Unused method argument: `dt`

(ARG002)